core/src/trezor/ui/layouts/tt/__init__.py

```diff
@@ -162,6 +162,16 @@ def confirm_multisig_warning() -> Awaitable[None]:
     )
 
 
+if not utils.BITCOIN_ONLY:
```

Comment:
can we move both these things to the big dedicated "if not BITCOIN_ONLY" section far below, together with the other altcoin stuff?